documents/meetup-starter-kit.md

```diff
@@ -0,0 +1,82 @@
+## Structure/Format
+
+Two to three presenters has been found to be best by multiple meetup coordinators.
```

Comment:
Reversing this sentence makes it a little more approachable. Something like "Meetup organizers have recommended two to three presenters as a good balance."